MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -933,6 +933,18 @@ private void PlatformSetViewport(ref Viewport value)
         }
 
         // Only implemented for DirectX right now, so not in GraphicsDevice.cs
+        public void SetRenderTarget(RenderTarget2D renderTarget, int arraySlice)
+        {
+            if (!GraphicsCapabilities.SupportsTextureArrays)
+                throw new InvalidOperationException("Texture arrays are not supported on this graphics device");
+
+            if (renderTarget == null)
+                SetRenderTarget(null);
+            else
+                SetRenderTargets(new RenderTargetBinding(renderTarget, arraySlice));
+        }
+
+        // Only implemented for DirectX right now, so not in GraphicsDevice.cs
         public void SetRenderTarget(RenderTarget3D renderTarget, int arraySlice)
```

Comment:
Actually... `TextureCube`, and `Texture2D` with `arraySize > 1`, **are** basically the same thing. The only difference, in Direct3D anyway, is in the `BindFlags`. (And same goes for `RenderTargetCube` and `RenderTexture2D`).